tests/hpc/ganglia_client.pm

```diff
@@ -1,23 +1,18 @@
 # SUSE's openQA tests
 #
-# Copyright 2018 SUSE LLC
+# Copyright 2018-2022 SUSE LLC
 # SPDX-License-Identifier: FSFAP
 
 # Summary: Ganglia Test - client
 #   Acts as client node, which publishes data to the server via gmetric command
-# Maintainer: soulofdestiny <mgriessmeier@suse.com>
-# Tags: https://fate.suse.com/323979
```

Comment:
I suggest to keep that. Even if fate is not actively used anymore it still acts as a reference explaining why we have test modules at all

Reply:
*this change is ready for review*